columnflow/plotting/plot_util.py

```diff
@@ -379,13 +356,10 @@ def use_flow_bins(
     return h_out
 
 
-def apply_density_to_hists(hists: dict, density: bool | None = False) -> dict:
+def apply_density(hists: dict) -> dict:
     """
     Scales number of histogram entries to bin widths.
     """
-    if not density:
-        return hists
-
```

Comment:
I don't understand this change. I much prefer to keep this in here such that all the `apply_<parameter>` functions are just one-liners such that resulting plot functions are as compact as possible. Especially since I already have quite a lot of custom plot functions which I'd now need to fix accordingly.

Reply:
Imho, calling a method even when it's clear it should not have an effect and thus, not be called in the first place is not a pattern that I would generally use.

However, I would include it again for backwards compatibility to not break existing user code as you pointed out.